docs/04-testing-emails.md

```diff
@@ -0,0 +1,5 @@
+# Testing emails
+
+## How do we do this
+
+I don't know. With [Litmus](https://www.litmus.com/pre-send-testing/)?
```

Comment:
Shall we leave this out until we do know?

Reply:
I think I was juts hungry before lunch. I've updated it to be a bit less sassy and more useful.